run_dir/static/js/running_notes.js

```diff
@@ -199,7 +202,9 @@ $('#rn_category ~ ul > li > button').on('click', function(){
 $('.rn-categ button').click(function(e){
     e.preventDefault();
     var was_selected = $(this).hasClass('active');
-    $('.rn-categ button').removeClass('active');
+    if(was_selected){
+        $(this).removeClass('active');
+    }
     if(!was_selected){
```

Comment:
```suggestion
    else{
```
Since we check the opposite condition in the if above